### PR TITLE
Merge changes to master

### PR DESCRIPTION
**Pull request by Linda-Editor**
Updated /office/dev/add-ins/develop/**create-addin-commands** at Elizabeth's request

### CODE REVIEW

**Comment by Linda-Editor** [comment]
@ElizabethSamuel-MSFT  I've made the changes to the Create add-ins doc; please merge the changes to master. I'll close the other bug.

File: reference/manifest/createaddincommands.md [hunk 1]
```diff
@@ -537,26 +459,25 @@ The following shows an example of how to use the **Resources** element. Each res
...
-## Additional resources
 <a name="bk_addresources"> </a>
 
--  [Add-in commands for Excel, Word and PowerPoint (Preview)](http://msdn.microsoft.com/library/5c898275-e47f-4e97-97df-098b769620d6%28Office.15%29.aspx)
-    
-  
--  [Define add-in commands in your Outlook add-in manifest](http://msdn.microsoft.com/library/4d57d987-cb49-484d-8a54-a55c2850c2aa%28Office.15%29.aspx)
+## See also
+
+-  [Add-in commands for Excel, Word, and PowerPoint](https://docs.microsoft.com/en-us/office/dev/add-ins/design/add-in-commands)
```
Comment (ElizabethSamuel-MSFT) [inline reference/manifest/createaddincommands.md]: For future reference, don't include "en-us" in the URL since that breaks loc navigation

Comment (Linda-Editor) [reply reference/manifest/createaddincommands.md]: Thanks.

**Comment by lindalu-MSFT** [comment]
@Linda-Editor There's a merge conflict, can you close this PR, fix it, and then open a new one?

**Comment by Linda-Editor** [comment]
Closing this PR because Elizabeth deleted this duplicate topic.

**Pull request closed without merging**